lib/scompler/kafka/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Scompler
   module Kafka
-    VERSION = '0.1.2'
+    VERSION = '0.1.2.1'
```

Comment:
Why do we need a fourth digit? I think it better use standard like semantic versioning: https://semver.org/.

Reply:
So:
```suggestion
    VERSION = '0.1.3'
```

Reply:
0.1.3 already exist. Changed to latest 0.1.5.